navigation/navigation-common/src/androidMain/kotlin/androidx/navigation/NavDeepLinkRequest.android.kt

```diff
@@ -20,33 +20,42 @@ import android.net.Uri
 import androidx.annotation.RestrictTo
 import java.lang.StringBuilder
 
+public actual typealias DeepLinkUri = Uri
```

Comment:
Since we do not compile `androidMain` at all, I do not understand how the information about these implementations (even typealias + inlines) will be delivered to the application.

So I doubt that it will work if it's defined in the same module that we binary-redirecting